### PR TITLE
Add parameter http endpoint under ensemble

### LINKED ISSUE #793: Add schema and entry point for parameters
Similarly to realizations and responses, it might be useful to expose parameters as a schema in the public api. For instance <ensemble_ref>/<parameter_ref> - > 
An example could look like but are not limited by:
```
{
   parameter_name: <name>
   realizations : [
     "realization_ref": <realization_idx>  
     "data_ref: <key>
  ]
}
```

### PR DESCRIPTION
**Pull request by ManInFez**
**Issue**
Resolves #793

### CODE REVIEW

File: ert_shared/storage/storage_api.py [hunk 1]
```diff
@@ -288,24 +286,73 @@ def ensemble_schema(self, ensemble_id):
...
+            "prior": {
+                "function": prior.function,
+                "parameter_names": prior.parameter_names,
+                "parameter_values": prior.parameter_values,
+            }
+            if prior is not None
+            else {},
+        }
+
+    def parameter(self, ensemble_id, parameter_def_id):
```
Comment (oysteoh) [inline ert_shared/storage/storage_api.py:318]: i would prefer naming like get_parameter since this is an api on top of the database

Comment (ManInFez) [reply ert_shared/storage/storage_api.py:318]: As of now the naming convention has mainly been like this At some point, we have to go over the storage_api and write a consistent api, and I agree that a get_ prefix might be nicer to read.

Comment (sondreso) [reply ert_shared/storage/storage_api.py:318]: Create separate issue and merge as is?

Comment (ManInFez) [reply ert_shared/storage/storage_api.py:382]: I would say merge as is and it will be part of the code cleanup issue :)

Comment (sondreso) [reply ert_shared/storage/storage_api.py:382]: Added here: https://github.com/equinor/ert/issues/819

**Pull request merged**